User documentation branch

aeldesoky opened the pull request:
Added the markdown user documentation. help.js converts the markdown to html and then append it to the body in help.twig. Let me know if you want me to change anything.

This is related to #22 .

Commit:
Added the markdown user documentation

Commit:
changed stuff

Commit:
tiny little change

brandon1024 commented on views/help.twig:
```diff
@@ -0,0 +1,21 @@
+
+{% block scripts %}
+  {{ parent() }}
+  <script type="text/javascript" src="./routes/help.js"></script>
```
I don't think you meant to serve the route file here hehe ;) You can remove this block, it's not needed :)

brandon1024 replied:
And could you move the stylesheet block above the body block? Just to keep consistent with the other twig files :)

brandon1024 commented on package.json:
```diff
@@ -13,13 +13,15 @@
     "express": "~4.15.5",
     "express-session": "^1.15.6",
     "font-awesome": "^4.7.0",
+    "fs": "0.0.1-security",
```
You don't need to pull this in as a dependency. `fs` is included with Node :)

brandon1024 commented on public/stylesheets/help.sass:
```diff
@@ -0,0 +1,5 @@
+.h1
```
You can leave these out, they don't change anything :)

brandon1024 commented on documents/help.md:
```diff
@@ -0,0 +1,21 @@
+## **About App Name** ##
+- - - -
+Oat cake dragée caramels dessert cotton candy gummi bears. Icing I love biscuit. Oat cake chocolate cake bonbon caramels sweet. Sesame snaps jelly beans pudding sweet dessert sugar plum sugar plum I love. Tiramisu chocolate cake marzipan pudding tiramisu oat cake icing I love. Cupcake bear claw dragée biscuit topping. Croissant I love gingerbread pie dessert wafer tart marshmallow brownie. I love ...
```
LOL these are great 😆

brandon1024 reviewed (changes requested):
Overall awesome! Just a couple of minor tweaks :)

Commit:
minor tweaks

aeldesoky commented:
@brandon1024 Removed fs from dependencies, fixed help.twig, removed the stuff in help.sass cause it wasn't doing anything, and removed its link from help.twig. I am going to leave the sass file in case we decide later on that we want to style something specific in the documentation
